app/models/container.rb

```diff
@@ -1,9 +1,11 @@
 class Container < ApplicationRecord
   include SupportsFeatureMixin
   include NewWithTypeStiMixin
+  include ContainerArchivedMixin
 
   has_one    :container_group, :through => :container_definition
   belongs_to :ext_management_system, :foreign_key => :ems_id
+  belongs_to :old_ext_management_system, :foreign_key => :old_ems_id, :class_name => 'ExtManagementSystem'
```

Comment:
@zeari @Fryguy should this belong to the mixin as well? (Is it even possible? Does it make sense? ...I am asking because I see that the mixing depends on it and it's repeated in all the classes).